src/shared/utils.ts

```diff
@@ -47,9 +47,10 @@ export function ensureArray<T>(value: T | T[]): T[] {
 export function undefinedIfLengthZero<T extends any[]>(arr: T) {
   return arr.length ? arr : undefined;
 }
-type NetworkMap<T> = Record<NetworkModes, T>;
-export function whenNetwork(mode: NetworkModes) {
-  return <T extends NetworkMap<unknown>>(networkMap: T) => networkMap[mode] as T[NetworkModes];
+type NetworkMap<T> = Record<BitcoinNetworkModes, T>;
+export function whenNetwork(mode: BitcoinNetworkModes) {
+  return <T extends NetworkMap<unknown>>(networkMap: T) =>
+    networkMap[mode] as T[BitcoinNetworkModes];
```

Comment:
Can you revert this change, sometimes we need the 2 network map. The function you're looking for exists https://github.com/leather-wallet/extension/blob/b926b27c50269d9f678e82a3aea0e1cdcc695375/src/app/common/utils.ts#L267-L269

Reply:
I may not have made any changes to this, but it's possible that I inadvertently merged something incorrectly.